libs/coin-modules/coin-aptos/src/logic/processRecipients.ts

```diff
@@ -0,0 +1,56 @@
+import { InputEntryFunctionData } from "@aptos-labs/ts-sdk";
+import { Operation } from "@ledgerhq/types-live";
+import { Operation as APIOperation } from "@ledgerhq/coin-framework/api/types";
+import { AptosAsset } from "../types/assets";
+import {
+  BATCH_TRANSFER_TYPES,
+  COIN_TRANSFER_TYPES,
+  DELEGATION_POOL_TYPES,
+  FA_TRANSFER_TYPES,
+} from "../constants";
+import { compareAddress } from "./getCoinAndAmounts";
+
+export function processRecipients(
+  payload: InputEntryFunctionData,
+  address: string,
+  op: Operation | APIOperation<AptosAsset>,
+  function_address: string,
+): void {
+  // get recipients buy 3 groups
+  if (
+    (COIN_TRANSFER_TYPES.includes(payload.function) ||
+      DELEGATION_POOL_TYPES.includes(payload.function)) &&
+    payload.functionArguments &&
+    payload.functionArguments.length > 0 &&
+    typeof payload.functionArguments[0] === "string"
+  ) {
+    // 1. Transfer like functions (includes some delegation pool functions)
+    op.recipients.push(payload.functionArguments[0].toString());
+  } else if (
```

Comment:
instead of using an if/else approach, I think it would be cleaner if you just do something like:
```
if (cond) {
  ...;
  return;
}

if (cond) {
  ...;
  return;
}

...

op.recipients.push(function_address);
```

Reply:
this is moved code, i don't want to mess up in this PR